### PR TITLE
Adds a section to launch Spark job using BeakerX Magic

### PR DESCRIPTION
**Pull request by farhan5900**
## Jira Ticket
https://jira.mesosphere.com/browse/DCOS-57952

## Description of changes being made
- Adds a section to launch Spark Job using BeakerX Magic

### CODE REVIEW

File: pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md [hunk 1]
```diff
@@ -215,3 +215,23 @@ count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduc
 print("Pi is roughly %f" % (4.0 * count / n))
 spark.stop
 ```
+
+### Using BeakerX Magic
```
Comment (akirillov) [inline pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md]: I'd suggest renaming it to `Using BeakerX Kernel` and adding something like that as the first line below the heading:
```
BeakerX Kernel supports Spark magic which allows to configure and start Spark Session via UI.
```

File: pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md [hunk 2]
```diff
@@ -215,3 +215,23 @@ count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduc
...
+```scala
+%%spark --start
+```
+
+```scala
+val count = spark.sparkContext.parallelize(1 to 10000).sum()
+count
+```
+
+If you want to use HDFS as a storage, you have to add following line in a first cell of the notebook.
```
Comment (akirillov) [inline pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md]: It's better to emphasize the importance of this step for History Server:
```
If you want to use HDFS for storage or as event log directory for Spark History Server, HDFS configuration files should be added to the classpath. To make HDFS configuration available for the notebook add the following line in the first cell of the notebook:
```

File: pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md [hunk 3]
```diff
@@ -215,3 +215,27 @@ count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduc
 print("Pi is roughly %f" % (4.0 * count / n))
 spark.stop
 ```
+
+### Using BeakerX Kernel
+[BeakerX Kernel](http://beakerx.com/) supports Spark Magic, which allows you to configure and start a Spark session via the UI.
+
+To use BeakerX Spark Magic, open a `Scala Notebook` and put the following in a seperate code cells:
```
Comment (samvantran) [inline pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md]:
```suggestion
To use BeakerX Spark Magic, open a `Scala Notebook` and put the following in a separate code cells:
```

File: pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md [hunk 4]
```diff
@@ -215,3 +215,27 @@ count = spark.sparkContext.parallelize(range(1, n + 1), partitions).map(f).reduc
...
+%%spark --start
+```
+
+
+```scala
+val count = spark.sparkContext.parallelize(1 to 10000).sum()
+count
+```
+
+If you want to use HDFS as storage or as an event log directory for a Spark History Server, HDFS configuration files should be added to the classpath. To make HDFS configuration available for the notebook, add the following line in a first cell of the notebook,
```
Comment (samvantran) [inline pages/mesosphere/dcos/services/data-science-engine/1.0.0/supported-kernels/index.md]:
```suggestion
If you want to use HDFS as storage or as an event log directory for a Spark History Server, HDFS configuration files should be added to the classpath. To make HDFS configuration available for the notebook, add the following line in the first cell of the notebook:
```

**Pull request merged**